cpp/src/jit/type.cpp

```diff
@@ -68,20 +68,18 @@ const void* get_data_ptr(scalar const& s)
   return type_dispatcher<dispatch_storage_type>(s.type(), get_data_ptr_functor{}, s);
 }
 
-std::string get_type_name(data_type type)
+std::string get_jit_type_name(data_type type)
```

Comment:
TODO: I need to confirm if this function can be removed entirely, in favor of using `cudf::type_to_name` everywhere.

Reply:
Following up: I removed this function in favor of `cudf::type_to_name` in f6939e6. Tests passed locally.